AirCasting/Utils/Strings.swift

```diff
@@ -1,302 +1,436 @@
+                                                                     comment: "")
+    }
+    
     enum Thresholds {
-        static let veryHigh: String = LocalizedStringKey("Max").stringKey
-        static let high: String = LocalizedStringKey("High").stringKey
-        static let medium: String = LocalizedStringKey("Medium").stringKey
-        static let low: String = LocalizedStringKey("Low").stringKey
-        static let veryLow: String = LocalizedStringKey("Min").stringKey
-    }
-
+        static let veryHigh: String = NSLocalizedString("Max",
+                                                        comment: "")
+        static let high: String = NSLocalizedString("High",
+                                                    comment: "")
+        static let medium: String = NSLocalizedString("Medium",
+                                                      comment: "")
+        static let low: String = NSLocalizedString("Low",
+                                                   comment: "")
+        static let veryLow: String = NSLocalizedString("Min",
+                                                       comment: "")
+    }
+    
     enum WifiPopupView {
-        static let wifiPlaceholder: String = LocalizedStringKey("Wi-Fi name").stringKey
-        static let passwordPlaceholder: String = LocalizedStringKey("Password").stringKey
-        static let connectButton: String = LocalizedStringKey("Connect").stringKey
-        static let passwordTitle: String = LocalizedStringKey("WiFi network name & password:").stringKey
-        static let nameAndPasswordTitle_1: String = LocalizedStringKey("Password for ").stringKey
-        static let nameAndPasswordTitle_2: String = LocalizedStringKey(" network:").stringKey
-        static let differentNetwork: String = LocalizedStringKey("Connect to a different WiFi network.").stringKey
-    }
-
+        static let wifiPlaceholder: String = NSLocalizedString("Wi-Fi name",
+                                                               comment: "")
+        static let passwordPlaceholder: String = NSLocalizedString("Password",
+                                                                   comment: "")
+        static let connectButton: String = NSLocalizedString("Connect",
+                                                             comment: "")
+        static let passwordTitle: String = NSLocalizedString("WiFi network name & password:",
+                                                             comment: "")
+        static let nameAndPasswordTitle_1: String = NSLocalizedString("Password for ",
+                                                                      comment: "It consists of few parts. Please consider them together. Whole text is following: Password for network %@")
+        static let nameAndPasswordTitle_2: String = NSLocalizedString(" network:",
+                                                                      comment: "")
```

Comment:
This is not enough to put a comment here. It won't work for languages that don't put the subject at the end of the sentence in this case. Can be worked around by providing an empty `nameAndPasswordTitle_2`, but this is not how it should be done. Use format string instead (`"Password for $@ network:"`).